bower.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "polymerfire",
-  "version": "0.9.4",
+  "name": "polymerfire-extended",
+  "version": "0.9.5",
   "authors": [
```

Comment:
:) Thanks a lot for the guidance (My first pull request)

I think it is ok now